src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -477,7 +477,7 @@ private void fetchFromSdkManager() throws IOException, InterruptedException {
 
     private void downloadSysrootZip(String arch) throws IOException {
         Logger.logInfo("Downloading sysroot zip...");
-        String sysrootZip = Strings.substitute(ARCH_SYSROOT_URL, Map.of("arch", arch));
+        String sysrootZip = Strings.substitute(ARCH_SYSROOT_URL, Map.of("arch", arch, "version", Constants.DEFAULT_SYSROOT_VERSION));
```

Comment:
Isn't this going to download the new version if there is an old version installed?

Reply:
No, it won't, there is a check that looks for `~/.gluon/substrate/sysroot/aarch64` installed, and if it is there, there is no check of possible versions, since we don't include the version number.